WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -114,10 +115,20 @@ class PostListFragment : Fragment() {
                     STANDARD -> {
                         recyclerView?.removeItemDecoration(itemDecorationCompactLayout)
                         recyclerView?.addItemDecoration(itemDecorationStandardLayout)
+                        (recyclerView?.layoutParams as? MarginLayoutParams)?.let {
+                            it.marginStart = nonNullActivity.resources
+                                    .getDimensionPixelSize(R.dimen.post_list_content_margin)
+                            it.marginEnd = nonNullActivity.resources
+                                    .getDimensionPixelSize(R.dimen.post_list_content_margin)
+                        }
                     }
                     COMPACT -> {
                         recyclerView?.removeItemDecoration(itemDecorationStandardLayout)
                         recyclerView?.addItemDecoration(itemDecorationCompactLayout)
+                        (recyclerView?.layoutParams as? MarginLayoutParams)?.let {
+                            it.marginStart = 0
```

Comment:
Will it make sense to have an `R.dimen.post_list_content_margin_compact` even if it's zero?

Reply:
Good call - fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9798/commits/db959aff4d6adbcdfa807694a36fe25d12985c42